charters/DataProcessingService/DataProcessingService.md

```diff
@@ -0,0 +1,51 @@
+# [Data Processing Service](mailto:pipelines-team@data.humancellatlas.org)
+
+## Description
+The data processing service is a service that executes automated transformation to scientific data through cloud-based execution of scientific pipelines. It can be used in any instantiation of the Data Coordination Platform (DCP). 
+
+## Objectives
+The data processing service aims to develop the infrastructure to run transformative scientific analysis pipelines reliably and at-scale for an instantiation of the DCP. The team working on the service will also operate the infrastructure for the Human Cell Atlas. 
```

Comment:
I got a bit confused by "for an instantiation of the DCP". It makes me think you're developing this for a specific DCP. I would scratch that part of the objective: 

The data processing service aims to develop the infrastructure to run transformative scientific analysis pipelines reliably and at-scale for ~an instantiation of~ the DCP. The team working on the service will also operate the infrastructure for the Human Cell Atlas.


I think this conveys the same idea. Please let me know if I've misinterpreted this.

Reply:
Updated.